src/words/environment/WordsObject.java

```diff
@@ -21,6 +21,10 @@ public WordsObject(String objectName, WordsClass wordsClass, Position cell) {
 		this.actionQueue = new LinkedList<Action>();
 		this.properties = new HashMap<String, Property>();
 	}
+	
+	public void clearQueue() {
```

Comment:
This name could be clearer (i.e. clearActionQueue), since an object could, in the future, have more than one queue.